spec/models/merchant_spec.rb

```diff
@@ -27,17 +27,22 @@
       @invoice_item1 = InvoiceItem.create!(item: @dice, invoice: @invoice1, quantity: 13, unit_price: 10.99 )
       @invoice_item2 = InvoiceItem.create!(item: @cursed_object, invoice: @invoice2, quantity: 1, unit_price: 6.00 )
     end
-
+    
     it '?sorted=age should return ascending by when created' do
       sorted_age = Merchant.sort_by_age("age")
       expect(sorted_age).to eq([@liquor_store, @kozey_group, @macho_man])
     end
-
-    it '?status=returned should return merchants with items on returned invoice' do
-      flustered_merchants = Merchant.status_returned('returned')
-      expect(flustered_merchants).to eq([@macho_man, @liquor_store])
+    
+    it 'returns all merchants if ?sorted= is blank or invalid' do
```

Comment:
I personally like the readability on the it blocks better than the method/params. Good call.